Lib/typing.py

```diff
@@ -2287,12 +2302,15 @@ def get_type_hints(obj, globalns=None, localns=None, include_extras=False,
     """
     if getattr(obj, '__no_type_check__', None):
         return {}
+    Format = _lazy_annotationlib.Format
+    if format is None:
+        format = Format.VALUE
```

Comment:
I think most uses of this function are likely to end up accessing `annotationlib`. To avoid overhead from repeated `_lazy_annotationlib` calls (even going through your `_LazyAnnotionlib.__getattr__` function isn't free) and keep the code clean, you could probably just do

```suggestion
    annotationlib = _lazy_annotationlib._annolib
```

or, if you like my suggestion at line 179

```suggestion
    annotationlib = _lazy_annotationlib()
```

here and then leave the rest of the function as it is on `main`

Reply:
The first suggestion doesn't work because `_annolib` is not guaranteed to exist yet. The second is likely slower; see above.